### PR TITLE
Fix hidden input prompt

### PR DESCRIPTION
**Pull request by autero1**
This PR fixes the issue with hidden prompt for input: https://github.com/gruntwork-io/terragrunt/issues/585

### CODE REVIEW

**Comment by autero1** [comment]
Tests pass and prompt is visible.
screen shot 2018-10-16 at 15 13 18

File: shell/run_shell_cmd.go [hunk 1]
```diff
@@ -46,51 +44,44 @@ func RunTerraformCommandAndCaptureOutput(terragruntOptions *options.TerragruntOp
...
-		outToErr = true
+		outWriter = terragruntOptions.ErrWriter
 	}
 
+	stdout := io.MultiWriter(outWriter, &outBuf)
+	stderr := io.MultiWriter(errWriter, &outBuf)
+
 	cmd.Dir = terragruntOptions.WorkingDir
+	cmd.Stderr = stderr
+	cmd.Stdout = stdout
```
Comment (brikis98) [inline shell/run_shell_cmd.go]: Could you just set these directly to `io.MultiWriter` without the extra intermediate variables?

Comment (brikis98) [reply shell/run_shell_cmd.go]: Also, could you add a link to https://blog.kowalczyk.info/article/wOYk/advanced-command-execution-in-go-with-osexec.html so future maintainers can reference it later?

Comment (autero1) [reply shell/run_shell_cmd.go]: Done.

File: shell/run_shell_cmd.go [hunk 2]
```diff
@@ -46,51 +44,41 @@ func RunTerraformCommandAndCaptureOutput(terragruntOptions *options.TerragruntOp
...
 	cmdChannel := make(chan error)
 	signalChannel := NewSignalsForwarder(forwardSignals, cmd, terragruntOptions.Logger, cmdChannel)
 	defer signalChannel.Close()
 
-	err = cmd.Wait()
+	err := cmd.Wait()
 	cmdChannel <- err
 
-	return output, errors.WithStackTrace(err)
+	return string(outBuf.Bytes()), errors.WithStackTrace(err)
```
Comment (brikis98) [inline shell/run_shell_cmd.go]: Sorry, one more tiny thing: I think you can do `outBuf.String()`

Comment (autero1) [reply shell/run_shell_cmd.go]: Fixed.

**Comment by brikis98** [comment]
https://github.com/gruntwork-io/terragrunt/releases/tag/v0.17.1

**Pull request merged**